### DISCUSSION TITLE
Installation error

### DISCUSSION THREAD

**Discussion by kasheex** [CURENT/andes#544] [General]
please help me how to resolve this?
<img width="713" alt="Screenshot 2024-06-15 at 10 15 39 AM" src="https://github.com/CURENT/andes/assets/57295204/51d3408d-8d0b-4167-b5ac-b656d03bc158">

**Comment by cuihantao** [comment]
Your Kvxopt is likely installed by pip, not conda/mamba. Pip sometimes struggles to set the correct path for compiled libraries.

You can uninstall the Kvxopt install by pip and do it with conda/mamba.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: kasheex ***@***.***>
Sent: Saturday, June 15, 2024 9:18:18 AM
To: CURENT/andes ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [CURENT/andes] Installation error (Discussion #544)


please help me how to resolve this?
Screenshot.2024-06-15.at.10.15.39.AM.png (view on web)<https://github.com/CURENT/andes/assets/57295204/51d3408d-8d0b-4167-b5ac-b656d03bc158>

**Reply by kasheex** [reply]
I really appreciate for your quick response.
Actually I installed all the things with "pip" to get into the dev mode but it wasn't working. Now I've restarted from the beginning using "conda" and it worked.

**Comment by kasheex** [comment]
Dear Cui,

I want to install in `dev` mode which is not possible with mambaforge/coda as mentioned in the documentation. If I go through `pip`, I get the same error as mentioned above. I need to develop a model, so I need to install `Andes` in development mode. 

Could you please help me out?

**Reply by cuihantao** [reply]
Hi,

Mambaforge is deprecated and replaced by Minforge.

Make sure you install Minforge, *install all the dependencies using mamba*, and use `pip install -e .` in the andes source code folder for a dev install.
